test/csuite/incr_backup/main.c

```diff
@@ -865,14 +606,9 @@ main(int argc, char *argv[])
         rnd.v = seed;
 
     testutil_work_dir_from_path(home, sizeof(home), working_dir);
-    /* Put the backup directories as the same level as the home directory. */
-    testutil_snprintf(backup_check, sizeof(backup_check), "./%s.CHECK", home);
-    testutil_snprintf(backup_dir, sizeof(backup_dir), "./%s.BACKUP", home);
-    testutil_snprintf(backup_src, sizeof(backup_src), "./%s.BACKUP.SRC", home);
     printf("Seed: %" PRIu64 "\n", seed);
 
     testutil_recreate_dir(home);
```

Comment:
What is the directory structure that this test creates?

It seems like it's creating the database right under `WT_TEST.incr_backup`, and then also directories such as `BACKUP.1` and `CHECK.1` right in the directory from where the test was started. It's probably not a big deal, but so far all the tests where I used the testutil backup functions assume that all directories are created under `WT_TEST.incr_backup`, so that it is clear that they belong together and so that they can be cleaned up together using a single directory remove.

With the directory structure as it is, there is a potential that, let's say a crashed test would leave some directory behind that might interfere with a later test, and it may also interfere with running several tests concurrently (if they are run from the same working directory). So I think it would be cleaner if this test could also consolidate all new directories under `WT_TEST.incr_backup` such as in `timestamp_abort`.

Reply:
You are right. I hadn't noticed because it cleans up on completion (it isn't an abort test so it isn't intentionally killing processes). I will try to add that quickly. If it gets complicated, I may make a ticket to do later in the project as I want to get moving on the analysis tools.